src/test/java/uk/gov/register/presentation/functional/TestEntry.java

```diff
@@ -1,23 +1,49 @@
 package uk.gov.register.presentation.functional;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.google.common.base.Throwables;
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.io.IOException;
 import java.time.Instant;
 
 public class TestEntry {
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
No, I strongly think that we must not couple PresentationApplication class in here or anywhere. I agree we can extract `customObjectMapper()` in separate class and can reuse if required.

Also this ObjectMapper is configured with serialization feature `SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS` and customObjectMapper is not.